SDKv3_Examples/app/src/main/java/com/mappedin/sdkv3_examples/MainActivity.kt

```diff
@@ -170,6 +172,49 @@ class MainActivity : AppCompatActivity() {
                     mapView.cameraControlsManager.setRotation(180.0)
                     mapView.cameraControlsManager.setTilt(0.0)
 
+                    // label all locations to be light on dark
+                    mapView.labelAllLocations(
+                        options=MPIOptions.LabelAllLocations(
+                            appearance=MPIOptions.LabelAppearance.lightOnDark
+                        )
+                    )
+
+                    // create a multi-destination journey between 4 sample locations
+                    mapView.venueData?.locations?.let { locations ->
+                        if (locations.size < 8) return@let
+
```

Comment:
I definitely don't know any kotlin, what does return@let mean?

Reply:
`return` in this situation would be ambiguous... Are we returning out of the scope of the `?.let { .. }`, `runOnUiThread { ... }` or `onFirstMapLoaded() { ... }`? So Kotlin statically generates special `return` variations for each case I listed in the previous sentence. Thus, `return@let` returns out of the scope of the `let` lambda.